app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center,
+                 clean_power,
+                 average_rating,
+                 count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car):
+        clean_dif = abs(car.clean_mark - self.clean_power)
+        rat_dist = self.average_rating / self.distance_from_city_center
+        return round(
+            car.comfort_class * clean_dif * rat_dist
+            , 1
```

Comment:
Maybe place store this value  `car.comfort_class * clean_dif * rat_dist` in the variable to avoid return statement three lines long